### ISSUE TITLE
UserWarning: Implicit dimension choice for softmax has been deprecated.

### ISSUE THREAD

**Issue by kasimte** [linnaeushuang/pensieve-pytorch#4]
I am running `python pensieve_torch.py --model_type=1` and I am getting this UserWarning repeatedly:

```
/Users/Kasim/Projects/ml/pensieve-pytorch/Network.py:84: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  out=F.softmax(self.outputLayer(fcOutput))
```

**Comment by linnaeushuang** [comment]
Hi, Sorry for a delayed response.This warning will appear in the latest version of pytorch.you can use 
`out=torch.softmax(self.outputLayer(fcOutput),dim=-1)`

**Comment by kasimte** [comment]
Thanks, @linnaeushuang!
